lib/shiftzilla/group.rb

```diff
@@ -1,10 +1,17 @@
 module Shiftzilla
   class Group
-    attr_reader :id, :lead
+    attr_reader :name, :id, :lead, :components
 
     def initialize(ginfo)
-      @id    = ginfo['id']
-      @lead  = ginfo['lead']
+      @name        = "Group " + ginfo['id']
+      @id          = ginfo['id']
+      @lead        = ginfo['lead']
+      @components  = []
     end
+
+    def set_components(component_list)
+      @components += component_list
```

Comment:
This feels a little weird to me. "set_components" implies that we are do a 'write once' procedure, but the `+=` opens the door for being able to add more components later, and I don't think we want to do that.

Reply:
I'll change it to use `||=`